lifecycle-operator/controllers/lifecycle/keptntask/job_utils.go

```diff
@@ -138,3 +144,10 @@ func (r *KeptnTaskReconciler) generateJob(ctx context.Context, task *klcv1alpha3
 
 	return job, nil
 }
+
+func getServiceAccount(serviceAccount *klcv1alpha3.ServiceAccountSpec) error {
+	if serviceAccount == nil {
```

Comment:
should the service account be mandatory ?

Reply:
actually, no